python/lsst/ts/externalscripts/maintel/make_comcam_bias.py

```diff
@@ -0,0 +1,76 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["MakeComCamBias"]
+
+import yaml
+
+from lsst.ts.observatory.control.maintel.comcam import ComCam
+from ..base_make_bias import BaseMakeBias
+
+
+class MakeComCamBias(BaseMakeBias):
+    """ Take biases and construct a master bias SAL Script.
+
+    This class takes biases with LSSTComCam and constructs
+    a master bias calling the bias pipetask via OCPS.
+    """
+
+    def __init__(self, index=1):
+        super().__init__(
+            index=index,
+            descr="This class takes biases with LSSTComCam and constructs "
+                  "a master bias calling the bias pipetask via OCPS.",
+        )
+        self._comcam = ComCam(domain=self.domain, log=self.log)
+        self._instrument_name = 'LSSTComCam'
+
+    @property
+    def camera(self):
+        return self._comcam
+
+    @property
+    def instrument_name(self):
+        """String with instrument name for pipeline task"""
+        return self._instrument_name
```

Comment:
consider returning the string `"LSSTComCam"` instead of a private object. This tells readers that this is really a fixed property rather than something that could potentially be updated privately.